packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -660,3 +662,44 @@ You then need to associate the registered Inline Hook with a Custom Authorizatio
 
 > **Note:** Only one Inline Hook can be associated with each rule.
 
+## Troubleshooting
+
+This section covers what happens when a token inline hook flow fails either due to the external inline hook service returning an error object or not returning a successful response, or the inline hook patch fails.
+
+### When there is an error
+
+- When the external service returns an error object in the response, the entire token inline hook flow fails with no token generated.
+
+  **Who can see this error?** Administrators, developers, and end users. When the OAuth 2.0 requester receives the error, the developer can see that error if the client has the debug information. What the end user sees depends on how errors are handled within the client.
+
+  > **Note:** See the [error](/docs/reference/token-hook/#error) section on this page for more information on what to include in the error object of your response and what the OAuth 2.0 error includes that Okta returns to the requestor of the token.
+
+- When a hook command (for example, updating, adding, and deleting claims) can't be performed, the inline hook operation is skipped. The token is generated without any modification from the inline hook.
+
+  **Who can see this error?** Administrators
+
+- When the external service returns a response with any other HTTP status code besides `200`, the inline hook operation is skipped. The token is generated without any modification from the inline hook.
+
+  **Who can see this error?** Administrators
+
+- When there is a communication failure with the external service, the inline hook operation is skipped. The token is generated without any modification from the inline hook.
+
+  **Who can see this error?** Administrators
+
+### Common Errors
+
+> **Note:** See the [Troubleshooting](/docs/concepts/inline-hooks/#troubleshooting) section in the Inline Hooks concept piece for more information on the events related to Inline Hooks that the Okta System Log captures.
+
+The `commands` array should only contain commands that can be applied to the requested tokens. For example, if only an ID token is requested, the `commands` array shouldn't contain commands of the type `com.okta.access.patch`.
```

Comment:
Based on Vadym's comment, this error will also result in the operation being skipped. I'd move it into the bullet points below as:
- Using an Invalid command. For example, if only an ID token is requested, the `commands` array shouldn't contain commands of the type `com.okta.access.patch`.